1-Introduction/3-fairness/translations/README.es.md

```diff
@@ -19,61 +19,61 @@ En esta lección, será capaz de:
 
 ## Prerrequisitos
 
-Como un prerrequisito, por favor toma el Path de aprendizaje "Responsible AI Principles" y mira el video debajo en el tema:
+Como un prerrequisito, por favor toma el curso "Responsible AI Principles" y mira el vídeo debajo sobre el tema:
```

Comment:
The translation for Learning Path in Microsoft Learn is _ruta de aprendizaje_

Reply:
I agree, that was a mistake on my part